src/types/emr/specimenDefinition/specimenDefinition.ts

```diff
@@ -85,8 +84,8 @@ export interface SpecimenDefinition {
   slug: string;
   derived_from_uri: string | null;
   status: Status;
-  description: string | null;
-  type_collected: Code | null;
+  description: string;
+  type_collected: Code;
   patient_preparation: Code[] | null;
```

Comment:
```suggestion
  patient_preparation?: Code[];
```

<img width="877" alt="image" src="https://github.com/user-attachments/assets/343e275a-06f5-4cd8-8b3b-2b53b9e7fa71" />